src/lms-ds-loader/.pylintrc

```diff
@@ -0,0 +1,14 @@
+[MASTER]
+ignore = .venv
+
+[MESSAGES CONTROL]
+disable =
+    C0330,
+    C0326,
+    fixme,
+    line-too-long,
+    missing-function-docstring,
+    missing-module-docstring
+
+[format]
+max-line-length = 88
```

Comment:
.flake8 says 110 and .pylint says 88

Reply:
black defaults to 88

Reply:
I've changed both to 110. There are some lines that Black can't reformat: those with long strings. I don't want flake/pylint to complain about those longer lines, up to 110 characters.